colab-coop/hello-voter
# Login bug

to test:

* run backend locally
* point frontend to your local backend
* run app 

you should be able to see the app and attempt to login
